css.webpack.js

```diff
@@ -57,48 +35,29 @@ module.exports = (env) => {
           ]
         },
         {
+          // this inlines resources if they are less than 8KB
           test: /\.(png|jpe?g|gif|svg|eot|ttf|woff|woff2)$/i,
-          loader: 'url-loader',
-          options: {
-            limit: 8192,
-          },
+          type: 'asset'
         }
       ]
     },
     plugins: [
       // extracts CSS to a separate file
       new MiniCssExtractPlugin({
-        filename: env.development ? '../stylesheets/[name].css' : '../stylesheets/[name].[hash].css',
+        filename: env.development ? '../stylesheets/[name].css' : '../stylesheets/[name].[contenthash].css',
       }),
       // generates a RTL version of the emitted CSS files
-      // this is done only in production
-      ...(env.production || env.coverage ? [new WebpackRTLPlugin({
-        filename: '../stylesheets/rtl-[name].[contenthash].css'
+      // this is done only in production/coverage
+      ...(env.production || env.coverage ? [new RtlCssPlugin({
+        filename: '../stylesheets/rtl-[name].[fullhash].css'
       })] : []),
-      // css-loader generates unnecesary js files (but with .styl extension)
-      // the following will remove that
-      new ExcludeAssetsPlugin({
-        path: ['^.*\\.styl$']
-      }),
-      // manifest file
-      new ManifestPlugin({
-        fileName: 'css-manifest.json',
-        map: (file) => {
-          if (/rtl-.*\.css$/.test(file.path)) {
-            file.name = `rtl-${file.name}`;
-          }
-          return file;
-        }
-      }),
-      // node environment
-      new webpack.DefinePlugin({
-        'process.env': {
-          NODE_ENV: JSON.stringify(mode),
-        },
-      })
     ],
-    watch: env.watch_js,
-    devtool: env.development ? 'eval' : 'source-map',
+    optimization: {
+      minimizer: [
+        '...',
+        new CssMinimizerPlugin(),
```

Comment:
Explore if we can pass any options that might be helpful.

Reply:
@TheTrio Were you able to explore this?

Reply:
The defaults were good enough in my opinion so there wasn't much to configure. As for CSS Nano options, auto prefixing was the only one that felt necessary to enable but then I found out that stylus loader already does that. 

So no, not much there I think. Do you have some specific configuration in mind?